my_project/my_app/views2.py

```diff
@@ -0,0 +1,34 @@
+"""
```

Comment:
What's this file?

Reply:
This file is an example of business logic transitions.  It is separate than the `views.py`, which just shows off adding/modifying slots.